### ISSUE TITLE
Travis build for python 3.6 broken

### ISSUE THREAD

**Issue by de-vri-es** [fizyr/keras-retinanet#247]
After a bit of digging it seems that the line `import keras` is enough to make py.test.

Making travis run `python -c "import keras"` gave a bit more details on what is going wrong [[1]](https://travis-ci.org/fizyr/keras-retinanet/builds/334257186):
```
0.91s$ python -c 'import keras'
Using TensorFlow backend.
RuntimeError: module compiled against API version 0xc but this version of numpy is 0xb
ImportError: numpy.core.multiarray failed to import
ImportError: numpy.core.umath failed to import
ImportError: numpy.core.umath failed to import
2018-01-28 01:47:59.036924: F tensorflow/python/lib/core/bfloat16.cc:664] Check failed: PyBfloat16_Type.tp_base != nullptr 
/home/travis/.travis/job_stages: line 57:  3639 Aborted                 (core dumped) python -c 'import keras'
The command "python -c 'import keras'" exited with 134.
```

It seems there is a mismatch between the compiled versions of tensorflow and numpy that we are using on travis.

[1] https://travis-ci.org/fizyr/keras-retinanet/builds/334257186

**Comment by de-vri-es** [comment]
Fixed by #248

**Comment by de-vri-es** [comment]
`numpy>=1.14` didn't work because it triggered shell redirection. Fixed that in b6b1b593eef8ad85c2f18ccb3213316fc8a2dfaf (also for `opencv>=3.3.0`)
